plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -50,6 +48,26 @@
   const client = getClient()
   const taskId: Ref<Issue> = generateId()
 
+  $: _space = space
+  $: _parent = parent
+  $: if (object.status === '') {
+    getDefaultIssueStatusId(_space).then((statusId) => {
```

Comment:
Better to not mix promises and async/await and use one style. For platform it is preferred to use async/await